app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -56,7 +56,7 @@ public GameView(Context context, int screenX, int screenY, boolean isPlaying) {
         this.isPlaying = isPlaying;
         // Margin in px.
         screenMargin = (int) getResources().getDimension(R.dimen.fab_margin);
-        // Create droid.
+        // Droid Creating.
```

Comment:
nit: this comment does not really help, you can remove it

Reply:
Fixed